README.md

```diff
@@ -99,6 +101,45 @@ The following are runtimes which are no longer or not yet supported by this repo
 [3]: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/faas/faas-spans.md#outgoing-invocations
 [4]: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/faas/faas-metrics.md
 
+## (Experimental) Customized collector build
```

Comment:
These sections might be better in the `collector/README.md` instead, especially since that documents how to publish the resulting layer.

Reply:
Moved the section to the collector README and updated the link in the main README accordingly